SPECS/containerized-data-importer/cdi_containers_meta

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+TAGPREFIX=kubevirt
+LABELPREFIX=org.opensuse.kubevirt
```

Comment:
I don't think we need this file.

Reply:
The cdi_containers_meta file itself has been removed.